doc/source/tutorials/tutorial-level-5-migration.rst

```diff
@@ -185,14 +197,20 @@ The package will most likely have failed ``pre-commit`` hooks. We will manually
 
 Here, instead of fixing all errors at once, we will address each type of error one at a time.  For example, a branch called ``pre-commit-spelling`` may contain spelling fixes, while another branch, ``pre-commit-flake8-line`` fixes of line length errors raised by flake8.
 
-#. Sync with the ``upstream/migration`` branch:
+#. To get the ``migration`` branch from ``upstream/migration`` and track it, run the following commands:
+
+    .. code-block:: bash
+
+        $ git fetch upstream migration
+        $ git checkout --track upstream/migration
```

Comment:
do you need to write `--track upstream/migration`? 

The `migration` branch already exists at this point both in the remote and the local right?

Reply:
I also commented below on @bobleesj 's comment.  I like haing the `--track` for explicitness so I vote for it.